appengine/standard/firebase/firetactoe/rest_api.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstration of the Firebase REST API in Python"""
+
+# [START rest_writing_data]
+import base64
+import json
+import os
+import re
+import time
+import urllib
+
+
+from flask import request
+import httplib2
+from oauth2client.client import GoogleCredentials
+
+
+_FIREBASE_SCOPES = [
+    'https://www.googleapis.com/auth/firebase.database',
+    'https://www.googleapis.com/auth/userinfo.email']
+
+
+def _get_http(_memo={}):
+    """Provides an authed http object."""
+    if 'http' not in _memo:
+        # Memorize the authorized http, to avoid fetching new access tokens
+        http = httplib2.Http()
+        # Use application default credentials to make the Firebase calls
+        # https://firebase.google.com/docs/reference/rest/database/user-auth
+        creds = GoogleCredentials.get_application_default().create_scoped(
+            _FIREBASE_SCOPES)
+        creds.authorize(http)
+        _memo['http'] = http
+    return _memo['http']
+    
+def fb_put(path, value=None):
```

Comment:
Prefer avoiding abbreviations, for clarity.
Also - I always read `fb` as `facebook`